machine_data_model/nodes/connectors/opcua_connector.py

```
@@ -0,0 +1,440 @@
+        :return: True if the client is connected to the server
+        """
+        success = self._handle_task(self._async_connect())
+        return success
+
+    async def _async_connect(self) -> bool:
+        """
+        Async function which uses the asyncua library to connect to the OPC-UA server.
+
+        :return: True if the client is connected to the server
+        """
+        url = "opc.tcp://{}:{}".format(self.ip, self.port)
```

Ruff (0.12.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)